geonode/layers/views.py

```diff
@@ -1355,7 +1355,7 @@ def layer_replace(request, layername, template='layers/layer_replace.html'):
         else:
             errormsgs = []
             for e in form.errors.values():
-                errormsgs.append([escape(v) for v in e])
+                errormsgs.append(escape(v) for v in e)
```

Comment:
This is not equivalent. Instead of adding a new list to an existing list, this code will instead add the generator object itself. As an example, you can run
```python
>>> mylist = [["a", "b"], ["one", "two"]]
>>> sequence = [1, 2]
>>> mylist.append(item for item in sequence)
>>> mylist
[['a', 'b'], ['one', 'two'], <generator object <genexpr> at 0x7fce659ca2e0>]
>>> mylist.pop()
<generator object <genexpr> at 0x7fce659ca2e0>
>>> mylist.append([item for item in sequence])
>>> mylist
[['a', 'b'], ['one', 'two'], [1, 2]]
```